### DISCUSSION TITLE
Cannot start copilot "Error detected while processing function <lambda>2[1]..<SNR>29_STart[4]..copilot#agent#New:"

### DISCUSSION THREAD

**Discussion by ChausseBenjamin** [community/community#38114] [Copilot Conversations]
Whenever I try to use copilot, I get the following error:
```
Error detected while processing function <lambda>2[1]..<SNR>29_Start[4]..copilot#agent#New:
line   13:
E688: More targets than List items
Press ENTER or type command to continue
```

I am running arch linux and am using the `nodejs-lts-gallium` package to remain on version 16 of nodejs. I installed and maintain copilot using vim-plug.

**Comment by ossanna16** [comment]
Hi there @ChausseBenjamin and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by tpope** [comment]
What output do you get when you call `:!node --version`?

**Reply by ChausseBenjamin** [reply]
The output I received was the following:
```
node: error while loading shared libraries: libcui18n.so.72: cannot open shared object file: No such file or directory
```

Knowing this, I managed to research the issue and install the correct libcui18n version by running the following command: `sudo pacman -Sy brltty harfbuzz-icu icu`.
